src/scirpy/pl/_logoplots.py

```diff
@@ -0,0 +1,89 @@
+from collections.abc import Sequence
+from typing import Literal
+
+from logomaker import Logo, alignment_to_matrix
+
+from scirpy.get import airr as get_airr
+from scirpy.util import DataHandler
+
+
+@DataHandler.inject_param_docs()
+def logoplot_cdr3_motif(
+    adata: DataHandler.TYPE,
+    chains: Literal["VJ_1", "VDJ_1", "VJ_2", "VDJ_2"] | Sequence[Literal["VJ_1", "VDJ_1", "VJ_2", "VDJ_2"]] = "VDJ_1",
+    airr_mod="airr",
+    airr_key="airr",
+    chain_idx_key="chain_indices",
+    cdr3_col: str = "junction_aa",
+    to_type: Sequence[Literal["information", "counts", "probability", "weight"]] = "information",
+    pseudocount: float = 0,
+    background=None,
+    center_weights: bool = False,
+    plot_default=True,
+    **kwargs,
+):
+    """
+    A user friendly wrapper function for the logomaker python package.
+    Enables the analysis of potential amino acid motifs by displaying logo plots.
+    Subsetting of AnnData/MuData has to be performed manually beforehand (or while calling) and only cells with equal cdr3 sequence lengths are permitted.
+
+    Parameters
+    ----------
+    {adata}
+    chains
+        One or multiple chains from which to use CDR3 sequences
+    {airr_mod}
+    {airr_key}
+    {chain_idx_key}
```

Comment:
We typicalle allow the user to specify an `ax` object into which the plot is added. This allows the user to easily compose multi-panel plots, e.g.

```python
fig, ax =plt.subplots()
ir.pl.something (..., ax=ax)
```

Do you think this is possible with logomaker?

Reply:
I see.
I will have a look on your other graphical implementations, but as far as I'm concerned (by looking at the logomaker documentation with working examples) this should be possible. So the idea is that the user can access the ax object after using the function right? So it has to be returned at some point in the function call, right?

Reply:
it's mostly useful for figure composition, i.e. one can make a multi planel figure using
```python
fig, (ax1, ax2) = plt.subplots(2,1)
```

and then plot different things into `ax1` and `ax2`.